mappings.json

```diff
@@ -0,0 +1,300 @@
+{
+  "freewheel": {
```

Comment:
why put the name here?

Reply:
In future if we add support for any other adserver, that adserver might have different mapping. Here for freewheel we are using id but for dfp it maybe possible that we want category name. 

```
{
  "freewheel": {
    "mapping": {
      "IAB1-1":1
    }
  },
  "dfp": {
    "mapping": {
      "IAB1-1": "Food"
    }
  }
}
```

Translation module will automatically detect adserver in use and use appropriate mapping

Reply:
Discussed about this offline. Prebid will have default file for each adserver. As of now we support freewheel so it will be `freewheel-mapping.json`. 
PR update coming soon.

Reply:
instead could we not save different files for different ad servers?  incase publishers start adding their own this scheme may become problematic. That was our plan with PBS.

Reply:
this is an outdated diff.